File: pset1/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void){
    char ch=' ';
    int height; 
    int i;
    int j; 
    int o;
    int e;
    int f;
    // int c is for the loop of wanted gaps
    int c; 
    
    // GIVE NUMBER
    do {
        height = get_int("Put height: ");
    }
    while(height<1 || height>8  );
    
    //FOR LOOPS START
    for (i=0; i<height; i++)
    {
        for ( o=height-2; o>=i; o--)
        {
            putchar (ch);
        }
        for (j=0; j<=i; j++)
        {
            printf("#");
        }
    for(c=0; c<2; c++) {
        putchar(ch);
    }
    
       
     
        for(f=0; f<=i; f++){
          printf("#");
   }
  
  
  printf("\n");
        }
        
        
    }

File: sentimental-mario-more/mario.py
# TODO
from cs50 import get_int

height = get_int("Height: ")
while height > 8 or height < 1:
    height = get_int("Height: ")
for i in range(height):
        #PRINT HASHES
        print((height-i-1)*" "+(i+1)*"#", end="")
        #PRINT GAPS
        print("  ", end="")
        #PRINT HASHES OF OTHER 
        print((i+1)*"#")